apps/prairielearn/python/traverse.py

```diff
@@ -158,8 +158,8 @@ def traverse_and_replace(
                 result.append(f"</{tail_tag}>")
             if tail_text is not None:
                 result.append(tail_text)
-        else:
-            count_stack[-1] -= 1
+
+        count_stack[-1] -= 1
```

Comment:
Was this change automated? Just want to make sure this doesn't change the logic here.

Reply:
Yes. It noticed there was no breaks inside the loop, and so it rewrote the code.

Reply:
If this was automated then probably ok 👍